config.json

```diff
@@ -1735,17 +1735,17 @@
       {
         "groupId": "com.google.dagger",
         "artifactId": "dagger",
-        "version": "2.44.2",
-        "nugetVersion": "2.44.2",
+        "version": "2.43.2",
```

Comment:
Version downgrade

Reply:
I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2